config/ember-try.js

```diff
@@ -94,6 +94,22 @@ module.exports = function() {
             }
           }
         },
+        {
+          name: 'ember-classic',
+          env: {
+            EMBER_OPTIONAL_FEATURES: JSON.stringify({
+              // TODO: tests fail if optional feature application-template-wrapper is enabled
+              // 'application-template-wrapper': true,
```

Comment:
I've no idea why this is happening. But I don't think it's related to this PR. It might also not be a real-world problem but only be triggered by our tests. At least I'm not aware of any bug reports that may be related to this.

Reply:
Not sure which test failed because of this, but I would expect this to potentially cause failures in tests. Especially for modals or other wormholed stuff. 

The flag also affects tests, so with it enabled `await render(hbs\`<BsButton .../>\`);` would render the button wrapped in a `<div>` which is then the application root element. Something like `await click('*')` (we don't have this in the code base much anymore, but I previously used this to pick the first rendered element, whatever the tag name it has or class...) would not trigger the click on the button, but the wrapper.

For modals [`getDestinationElement()`])(https://github.com/kaliber5/ember-bootstrap/blob/master/addon/utils/dom.js#L63) would pick the test container as the wormhole destination element. But with the flag enabled, the wrapper div would become the application root element, thus the modal would be rendered *outside* of it, causing at least problems with events (see the other issue comment of today)

This is only an issue for tests though!